benchmark/libsodium/main.dart

```diff
@@ -119,5 +120,8 @@ main() {
   LibSodiumSHA256Benchmark.main();
   LibSodiumSHA512Benchmark.main();
   LibSodiumSHA512FFIOnlyBenchmark.main();
-  LibSodiumScryptPasswordStorageBenchmark.main();
+  if (Platform.isLinux) {
```

Comment:
what about windows ? ^^ maybe better to run them with try/catch and print "not supported on ..." if we have a catch